packages/rocket-auth-aws-infrastructure/src/lambdas/challenge-create.ts

```diff
@@ -0,0 +1,29 @@
+import { CognitoUserPoolEvent } from 'aws-lambda'
+import { SNS } from 'aws-sdk'
+
+const randomSecretCode = (min = 100000, max = 999999): string => {
+  return Math.floor(Math.random() * (max - min + 1) + min).toString()
+}
```

Comment:
Can you add a doc comment to this please?

Reply:
Done!